x-pack/plugins/observability_ai_assistant/public/components/chat/welcome_message.tsx

```diff
@@ -87,7 +87,7 @@ export function WelcomeMessage({
         className={fullHeightClassName}
       >
         <EuiFlexItem grow className={centerMaxWidthClassName}>
-          <EuiSpacer size={breakpoint === 'xl' ? 'l' : 'l' ? 'l' : 's'} />
+          <EuiSpacer size={breakpoint === 'xl' ? 'l' : breakpoint === 'l' ? 'l' : 's'} />
```

Comment:
Isn't it the same?
```suggestion
          <EuiSpacer size={breakpoint === 'xl' || breakpoint === 'l' ? 'l' : 's'} />
```

Reply:
Yes, agreed. Will do this instead. Nested ternaries are rarely pleasant to read.